.github/workflows/build.yml

```diff
@@ -35,8 +35,10 @@ jobs:
         pip install -e .
     - name: Install pytest
       run: |
-        mamba install -c conda-forge pytest
+        mamba install -c conda-forge pytest black
     - name: Test with pytest
       run: |
         pytest
-
+    - name: Lint with black
+      run: |
+        black .
```

Comment:
Should this be `black --check .`? The CI actually reformatting the code won't be helpful; we just want the build to fail if it's not already successfully blacked, right?

Reply:
yes, thank you; fixing ...